source/getting-started.html.md

```diff
@@ -0,0 +1,238 @@
+utilises the [`answer` schema][answer-schema].
+
+### Details
+
+```
+{
+  ...
+  "details": {
+    "body": "\n<div class=\"highlight-answer\">\n<p>The standard <abbr title=\"Value Added Tax\">VAT</abbr> rate is <em>20%</em></p>\n</div>\n\n<h2 id=\"vat-rates-for-goods-and-services\">\n<abbr title=\"Value Added Tax\">VAT</abbr> rates for goods and services</h2>\n\n<table>\n  <thead>\n    <tr>\n      <th>Rate</th>\n      <th>% of <abbr title=\"Value Added Tax\">VAT</abbr>\n</th>\n      <th>What the rate applies to</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <td>Standard</td>\n      <td>20%</td>\n      <td>Most goods and services</td>\n    </tr>\n    <tr>\n      <td>Reduced rate</td>\n      <td>5%</td>\n      <td>Some goods and services, eg children’s car seats and home energy</td>\n    </tr>\n    <tr>\n      <td>Zero rate</td>\n      <td>0%</td>\n      <td>Zero-rated goods and services, eg most food and children’s clothes</td>\n    </tr>\n  </tbody>\n</table>\n\n<p>The standard rate of <abbr title=\"Value Added Tax\">VAT</abbr> increased to 20% on 4 January 2011 (from 17.5%).</p>\n\n<p>Some things are exempt from <abbr title=\"Value Added Tax\">VAT</abbr>, eg postage stamps, financial and property transactions.</p>\n\n<p>The <a href=\"/vat-businesses/vat-rates\"><abbr title=\"Value Added Tax\">VAT</abbr> rate businesses charge</a> depends on their goods and services.</p>\n\n<p>Check the <a href=\"https://www.gov.uk/rates-of-vat-on-different-goods-and-services\">rates of <abbr title=\"Value Added Tax\">VAT</abbr></a> on different goods and services.</p>\n\n",
+    "external_related_links": []
+  },
+  ...
+}
+```
+
+The `details` field of a [`ContentItem`][content-item] is an object that
+contains data in a particular structure relevant for the type of content. This
+structure is described by the [schema](#document-type-and-schema-name), which
+itself is defined by the `schema_name` field.
+
+In the case of [Vat Rates][vat-rates-api] an [`answer` schema][answer-schema]
+is used. The `answer` schema  allows a `body` value, which contains the HTML
+for the page and a `external_related_links` value, which is an array that can
+be used to provide links to associated pages hosted off GOV.UK.
+
+### Links
+
+The `links` field is used to describe relationships between items of content.
+It is presented as an object where the key represents a link type and the
+value represents an array of [`LinkedContentItems`][linked-content-item]. A
+`LinkedContentItem` is effectively a condensed [`ContentItem`][content-item].
+
+The values that can be used for link types are defined within the
+[schema](#document-type-and-schema-name) associated with the content.
+This value describes the type of relationship between contnet. For example,
+with [Vat Rates][vat-rates-api] there is a relationship with a link type of
+`mainstream_browse_pages`. The purpose of this link type is to allow ability
```

Comment:
I'd suggest:
"The purpose of this link type is to enable linking to pages to help users find similar content that is suitable for a mainstream audience"